File: src/main/scala/comp354/pm/controller/ProjectController.scala
package comp354.pm.controller

import comp354.pm.model.Tables._
import comp354.pm.model.ProjectDAO

object ProjectController {

  def add(p: ProjectRow) = ProjectDAO.add(p)
  def get(u:UserRow) = ProjectDAO.get(u)
  def get(id: Int) = ProjectDAO.get(id)
}

File: src/edu/concordia/comp354/model2/controller/ProjectController.java
package edu.concordia.comp354.model2.controller;

import edu.concordia.comp354.model2.dao.ProjectDAO;
import edu.concordia.comp354.model2.model.Project;
import edu.concordia.comp354.model2.model.User;

import java.sql.SQLException;
import java.util.Collection;

public class ProjectController {

	private ProjectDAO projectDAO;

	public ProjectController(ProjectDAO projectDAO) {
		this.projectDAO = projectDAO;
	}

	public Project add(Project p) throws SQLException {
		return projectDAO.add(p);
	}

	public void delete(Project p) throws SQLException {
		projectDAO.delete(p);
	}

	public Collection<Project> get(User u) throws SQLException {
		return projectDAO.get(u);
	}

	public void update(Project p) throws SQLException {
		projectDAO.update(p);
	}
}
